DHIS2/dutyday/reporter.py

```diff
@@ -0,0 +1,119 @@
+import subprocess
+import datetime
+import sys
+import subprocess
+import json
+
+import paramiko
+
+
+#this script will be executed on local docker with vpn active
+def execute_command_on_remote_machine(host, command):
+    path_to_private_key = '/root/.ssh/id_rsa'  # ruta a tu archivo de clave privada SSH
+    private_key = paramiko.RSAKey.from_private_key_file(path_to_private_key)
+
+    client = paramiko.SSHClient()
+    client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+    client.connect(host, username="tomcatuser", pkey=private_key)
+
+    stdin, stdout, stderr = client.exec_command(command)
+    output = stdout.read().decode().strip()
+
+    client.close()
+
+    return output
+
```

Comment:
I get why its hard-coded in the script, as we both use the same docker image, but in a generic case the key path and the username should be either command line arguments or in the config.json file.

Given the intended users of the script its fine like this.